config/perf.rest.server.properties

```diff
@@ -0,0 +1,9 @@
+# rest server
+rest.server.blob.storage.service.factory=com.github.ambry.frontend.AmbryBlobStorageServiceFactory
```

Comment:
Not related to this PR.

Do we have documentation stating all the mandatory configs to start up a rest server or to start up a store server? For rest server, seems the mandatory configs are router.hostname and router.datacenter.name. This info, is however not very transparent for the first-time user.